tests/oclinter/paths-relative-leafref/openconfig-testcase-succeed.yang

```diff
@@ -0,0 +1,50 @@
+module openconfig-testcase-succeed {
```

Comment:
not sure i understand this test -- should it include a leafref type with a relative path?